netlify.toml

```diff
@@ -20,17 +20,10 @@ HUGO_ENABLEGITINFO = "true"
 command = "git submodule update --init --recursive --depth 1 && make deploy-preview"
 
 [context.branch-deploy]
-command = "git submodule update --init --recursive --depth 1 && make deploy-preview"
+command = "git submodule update --init --recursive --depth 1 && make non-production-build"
```

Comment:
When is the `branch-deploy` context used?

Reply:
The context was called `branch-deploy` before; I assume that's correct (I'm not touching that aspect).